loopy/transform/privatize.py

```diff
@@ -33,10 +34,11 @@
 .. currentmodule:: loopy
 
 .. autofunction:: privatize_temporaries_with_inames
+.. autofunction:: unprivatize_temporaries_with_inames
 """
 
 
-# {{{ privatize temporaries with iname
+# {{{ privatize temp oraries with iname
```

Comment:
```suggestion
# {{{ privatize temporaries with iname
```

Reply:
Fixed in 009a42435eea3a08c048de73b922481a3c4b1b96